src/main/java/net/sf/jabref/collab/ChangeDisplayDialog.java

```diff
@@ -64,6 +65,10 @@ public ChangeDisplayDialog(JFrame owner, final BasePanel panel,
         }
         tree = new JTree(root);
         tree.addTreeSelectionListener(this);
+        // Fix tree row height
+        FontMetrics metrics = tree.getFontMetrics(tree.getFont());
```

Comment:
Can these two lines be separated into a utility method, which gets `tree` passed? (At the other places, the appropriate other object; `table`, ...)